OpenSim/Tests/testIterators/testIterators.cpp

```diff
@@ -454,6 +558,7 @@ int main() {
         SimTK_SUBTEST(testComponentListNonConstWithConstIterator);
         SimTK_SUBTEST(testComponentListNonConstWithNonConstIterator);
         SimTK_SUBTEST(testComponentListComparisonOperators);
+        SimTK_SUBTEST(testComponentListRefs);
```

Comment:
without comments a more descriptive test name would be helpful, like: `testNestedComponentListConsistency` or something along those lines.